src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -63,8 +63,7 @@ const statusCell = (workflowStatuses, status) => {
     TooltipTrigger,
     {
       side: 'bottom',
-      type: 'light',
```

Comment:
This was the only place that used the "light" tooltip type. The differences between it and the standard tooltip were very subtle, so I decided to remove it. Removing the margin on the table here is to offset the additional padding from the standard tooltip.

Reply:
thanks for pointing this out... can we get @cahrens and UX to weigh in on this change?

Reply:
The change is so small, I don't feel a need to ask UX.

## Before
![Screenshot 2023-09-19 at 2 11 36 PM](https://github.com/DataBiosphere/terra-ui/assets/1156625/3a63f2f6-4dec-4da5-86a3-bffed396ddd8)

## After
![Screenshot 2023-09-19 at 2 11 00 PM](https://github.com/DataBiosphere/terra-ui/assets/1156625/0bc079e0-367c-45d4-ad31-313ad4e69d9e)

Reply:
add a before/after image to the PR?